packages/node/src/Session.ts

```diff
@@ -171,7 +171,16 @@ export class Session implements IHasSessionEventListener {
     });
 
     this.events.on(EVENTS.ERROR, () => this.internalLogout(false));
-    this.events.on(EVENTS.SESSION_EXPIRED, () => this.internalLogout(false));
+    this.events.on(EVENTS.SESSION_EXPIRED, () => {
```

Comment:
Logging out clears the storage, which deletes potential refresh tokens. In order to support the use case where one loads a session from storage, logs it in, performs some operation, and then lets it expire only to refresh it when needed again, we must stop clearing the storage on expiration.